t/058-tcp-socket.t

```diff
@@ -4,7 +4,7 @@ use Test::Nginx::Socket::Lua;
 
 repeat_each(2);
 
-plan tests => repeat_each() * 190;
+plan tests => repeat_each() * 190 + 6;
```

Comment:
This test plan is wrong since the `repeat_each` value may get changed in special test modes like the "check leak" test mode, in which case the test plan is just wrong. We should make the test plan irrelevant to the actual values of `repeat_each`, always. The fix is

```diff
diff --git a/t/058-tcp-socket.t b/t/058-tcp-socket.t
index 5dc1695e..0743dd73 100644
--- a/t/058-tcp-socket.t
+++ b/t/058-tcp-socket.t
@@ -4,7 +4,7 @@ use Test::Nginx::Socket::Lua;

 repeat_each(2);

-plan tests => repeat_each() * 190 + 6;
+plan tests => repeat_each() * 193;

 our $HtmlDir = html_dir;
```